pokemon/pokemon.py

```diff
@@ -0,0 +1,94 @@
+"""This module contains a two classes for creating pokemon."""
+
+from random import randint
+
+
+def repeat(m):
+    """Decorator-function allows decorated function to repeat m times."""
+
+    def inner(func_object):
+        def wrapper(*args, **kwargs):
+            print("Repeat ", m, " times")
```

Comment:
Python f-strings are a cool thing to use, you may want to look into them:
https://realpython.com/python-f-strings/
```suggestion
            print(f"Repeat  {m} times")
```